tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/Transports/Metrics/MetricsTransportTests.cs

```diff
@@ -0,0 +1,56 @@
+// <copyright file="MetricsTransportTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#if NETCOREAPP3_1
+
+using System.Collections.Concurrent;
+using Datadog.Trace.TestHelpers;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests
+{
+    public class MetricsTransportTests : TestHelper
+    {
+        public MetricsTransportTests(ITestOutputHelper output)
+            : base("TransportsTester", @"test\test-applications\regression", output, prependSamplesToAppName: false)
+        {
+        }
+
+        [Fact]
+        public void MetricsMatchAcrossTransports()
```

Comment:
```suggestion
        public void MetricsNotEmptyAcrossTransports()
```

Reply:
I will be updating this to be sure that the metrics match, this is just to start

Reply:
Removed and put metrics test side by side with runtime metrics